### PR TITLE
Resolves #1289

### LINKED ISSUE #1289: Page needs more details
This page describes the Set-SMBServerConfiguration command, including all the parameters you use to turn on/off SMB server properties. But the action of some of these properties is not explained. For example, what is a server announcement? What does validate target name do in practice, for example? 

This page is fine if you know what you are doing, configuration wise, but more detail would be useful.
Also rather than putting the details here, how about a more detailed page on these settings (and refer to that page here)?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fbf1abaf-c268-b76f-48b5-5f5723fecb83
* Version Independent ID: 0c17d9d3-bbc4-f467-b44c-d78bb0c7e370
* Content: [Set-SmbServerConfiguration (smbshare)](https://docs.microsoft.com/en-us/powershell/module/smbshare/set-smbserverconfiguration?view=win10-ps#feedback)
* Content Source: [https://cpubwin.visualstudio.com/_git/powershell?path=%2fdocset%2fwindows%2fsmbshare%2fset-smbserverconfiguration.md&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/_git/powershell?path=%2fdocset%2fwindows%2fsmbshare%2fset-smbserverconfiguration.md&version=GBlive&_a=contents)
* Product: **w10**
* Technology: **powershell-windows**
* Git Login: **andreabarr**
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Provide source for SMB information
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1289

### CODE REVIEW

**Comment by e0i** [comment]
cc @dariomws @get-itips @tseward @JohanFreelancer9

File: docset/windows/smbshare/Set-SmbServerConfiguration.md [hunk 1]
```diff
@@ -45,13 +45,15 @@ Set-SmbServerConfiguration [-AnnounceComment <String>] [-AnnounceServer <Boolean
 ```
 
 ## DESCRIPTION
-The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) Service configuration.
+The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) Service configuration. For get more information on SMB server and protocol specification, see [Overview of file sharing using the SMB 3 protocol in Windows Server](https://docs.microsoft.com/windows-server/storage/file-server/file-server-smb-overview) and [[MS-SMB2]: Server Message Block (SMB) Protocol Versions 2 and 3](https://docs.microsoft.com/openspecs/windows_protocols/ms-smb2/5606ad47-5ee0-437a-817e-70c366052962)
```
Comment (JohanFreelancer9) [inline docset/windows/smbshare/Set-SmbServerConfiguration.md:48]:
```suggestion
The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) Service configuration. For more information on SMB server and protocol specification, see [Overview of file sharing using the SMB 3 protocol in Windows Server](https://docs.microsoft.com/windows-server/storage/file-server/file-server-smb-overview) and [[MS-SMB2]: Server Message Block (SMB) Protocol Versions 2 and 3](https://docs.microsoft.com/openspecs/windows_protocols/ms-smb2/5606ad47-5ee0-437a-817e-70c366052962).
```

File: docset/winserver2012-ps/smbshare/Set-SmbServerConfiguration.md [hunk 2]
```diff
@@ -36,14 +36,16 @@ Set-SmbServerConfiguration [-AnnounceComment <String>] [-AnnounceServer <Boolean
 ```
 
 ## DESCRIPTION
-The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) server configuration.
+The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) server configuration. For get more information on SMB server and protocol specification, see [Server Message Block Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831795(v=ws.11)). For protocol specification, see [[MS-SMB2]: Server Message Block (SMB) Protocol Versions 2 and 3](https://docs.microsoft.com/openspecs/windows_protocols/ms-smb2/5606ad47-5ee0-437a-817e-70c366052962)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/Set-SmbServerConfiguration.md:39]:
```suggestion
The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) server configuration. For more information on SMB server and protocol specifications, see [Server Message Block Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831795(v=ws.11)). For protocol specification, see [[MS-SMB2]: Server Message Block (SMB) Protocol Versions 2 and 3](https://docs.microsoft.com/openspecs/windows_protocols/ms-smb2/5606ad47-5ee0-437a-817e-70c366052962).
```

File: docset/winserver2012r2-ps/smbshare/Set-SmbServerConfiguration.md [hunk 3]
```diff
@@ -44,13 +44,15 @@ Set-SmbServerConfiguration [-AnnounceServer <Boolean>] [-AsynchronousCredits <UI
 ```
 
 ## DESCRIPTION
-The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) server configuration.
+The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) server configuration. For get more information on SMB server and protocol specification, see [Server Message Block Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831795(v=ws.11)). For protocol specification, see [[MS-SMB2]: Server Message Block (SMB) Protocol Versions 2 and 3](https://docs.microsoft.com/openspecs/windows_protocols/ms-smb2/5606ad47-5ee0-437a-817e-70c366052962)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/Set-SmbServerConfiguration.md:47]:
```suggestion
The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) server configuration. For more information on SMB server and protocol specifications, see [Server Message Block Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831795(v=ws.11)). For protocol specification, see [[MS-SMB2]: Server Message Block (SMB) Protocol Versions 2 and 3](https://docs.microsoft.com/openspecs/windows_protocols/ms-smb2/5606ad47-5ee0-437a-817e-70c366052962).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**